channeldb/payment_control.go

```diff
@@ -171,14 +171,15 @@ func (p *PaymentControl) InitPayment(paymentHash lntypes.Hash,
 		// retrying the payment or return a specific error.
 		case err == nil:
 			if err := paymentStatus.initializable(); err != nil {
-				updateErr = err
+				updateErr = fmt.Errorf("initialize: %w", err)
```

Comment:
So If I revert this back to just: 
```go
updateErr = err
```

Then strangely, the `sendtoroute_multi_path_payment` starts to pass again 🤔

Reply:
Similarly, I also don't wrap the error down below, pushing this up as a commit to see how CI fares now.

Reply:
Not sure why, but when we wrap the error here, this error check fails, causing the whole itest to fail: https://github.com/lightningnetwork/lnd/blob/48a3d5633560d0f38c69e69b3a4bf520545c8590/routing/router.go#L2554